indico/modules/events/contributions/templates/management/_contribution_report.html

```diff
@@ -84,6 +87,18 @@
                                 {% endif %}
                             </td>
                             <td class="i-table">
+                                <a href="#" class="i-label"
+                                   title="{% trans %}Manage subcontributions{% endtrans %}"
+                                   data-href="{{ url_for('.manage_subcontributions', contrib) }}"
+                                   data-title="{% trans %}List of the subcontributions{% endtrans %}"
```

Comment:
I think this title looks rather strange. `Subcontributions of "{{ contrib_title }}"`?